packages/aws-cdk-lib/core/test/tag-aspect.test.ts

```diff
@@ -131,6 +144,23 @@ describe('tag aspect', () => {
     expect(res2.tags.renderTags()).toEqual([{ key: 'first', value: 'there is only 1' }]);
   });
 
+  test('Tags applied without priority gets mutating priority value', () => {
```

Comment:
I promise I'm not trying to be pedantic ;) 
```suggestion
  test('Tags applied without priority get mutating priority value', () => {
```